src/gui/guiiconprovider.h

```diff
@@ -30,30 +31,44 @@
 #ifndef GUIICONPROVIDER_H
 #define GUIICONPROVIDER_H
 
+#include "base/bittorrent/torrenthandle.h"
 #include "base/iconprovider.h"
 
 class QIcon;
 
-class GuiIconProvider : public IconProvider
+class GuiIconProvider: public IconProvider
 {
     Q_DISABLE_COPY(GuiIconProvider)
     Q_OBJECT
 
 public:
+
+    enum ThemeFlags
+    {
+        LightTheme = 1,
+        DarkTheme = 2,
+        SmallStatusIcons = 4,
+        LargeStatusIcons = 8,
```

Comment:
@evsh:  The files are not changed at all.  They are simply rendered without their transparent whitespace (in real-time).